Sources/DockProgress/DockProgress.swift

```diff
@@ -1,13 +1,34 @@
 import Cocoa
+import CoreVideo
 
 @MainActor
 public enum DockProgress {
-	private static var previousProgress: Double = 0
+    private static var t: Double = 0;
```

Comment:
Use a more descriptive name.
```suggestion
    private static var t = 0.0;
```

Reply:
Any ideas what a good name would be? Best I can come up with is `progressProgress` 🤣

Reply:
A bit verbose, but also descriptive: `elapsedTimeSinceLastRefresh`